api/v1/controllers/bots.js

```diff
@@ -86,12 +86,15 @@ module.exports = {
     const resultObj = { reqStartTime: new Date() };
     const reqObj = req.swagger.params;
     const seqObj = {};
+    const uiObj = {};
     try {
       for (const param in reqObj) {
         if (reqObj[param].value) {
           if (typeof (reqObj[param].value) === 'object' &&
```

Comment:
@shriramshankar I'm making that object to overwrite the value for UI in the response which is long binary data. I'm overwriting it at line 107.